less/release-9.less

```diff
@@ -0,0 +1,351 @@
+.flex-wrap {
```

Comment:
I think this should be added to solstice-assets:

https://github.com/EclipseFdn/solstice-assets/blob/master/less/_components/classes-grid.less

What do you think?

Reply:
I agree, will submit a patch for it.